framework/include/utils/MooseUtils.h

```diff
@@ -1006,13 +1006,7 @@ template <typename T>
 struct canBroadcast
 {
   static constexpr bool value = std::is_base_of<TIMPI::DataType, TIMPI::StandardType<T>>::value ||
-                                std::is_same<T, std::string>::value;
-};
-template <typename T>
-struct canBroadcast<std::vector<T>>
-{
-  static constexpr bool value = std::is_base_of<TIMPI::DataType, TIMPI::StandardType<T>>::value ||
-                                std::is_same<T, std::string>::value;
+                                TIMPI::Has_buffer_type<TIMPI::Packing<T>>::value;
```

Comment:
I thought this wasn't working? What changed?

Reply:
@roystgnr wanna do the honors?

Reply:
Nice. I started the parade of hindsight-stupidity with `std::pair` (oh of course you should be able to pack anything that is not a fixed type ... psych ... idiot!) and then we kept propagating it